src/patterns/cookies-page/index.md.njk

```diff
@@ -43,6 +43,15 @@ List all the cookies you’re using in the service. Divide the list into:
 - analytics cookies - cookies that let you collect analytics data to use within your own organisation
 - any other types of cookie you’re using
 
+You should also identify if each cookie is set on the server or client. 
+
+The result of your audit will guide your cookie policy and how the service should use a cookies page and cookie banner. 
+
+The [cookie banner component](/components/cookie-banner/) shows several options for using a cookie banner for services that:
+- only sets essential cookies
```

Comment:
Should these be 'set' rather than 'sets'?

Reply:
Okay, I had to look up the grammar rules on this as it was confusing me! "Sets" is used for present tense and third person. Otherwise it seems to always be "set"

So:
- You set cookies in your service
- Your service sets cookies in itself
- Your service had set cookies in itself